src/mgr/DaemonHealthMetricCollector.cc

```diff
@@ -77,12 +80,18 @@ class PendingPGs final : public DaemonHealthMetricCollector {
     if (osds.empty()) {
       return;
     }
-    check.summary = fmt::format("{} PGs pending on creation", value.n);
+    auto is_max_pg = value.n2;
+    check.summary =
+        fmt::format("{} PGs pending on creation {}",
+           value.n1,
+           (is_max_pg ? "(Withhold PGs creation due to overdose protection)" : ""));
```

Comment:
I think we should simply this message to be more user-friendly. Perhaps, explain what overdose protection means and what the user needs to do to fix this warning